playbooks/roles/node/tasks/main.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Assert OS is Ubuntu 18.04
+  assert:
+    that:
+      - ansible_distribution == 'Ubuntu'
+      - ansible_distribution_major_version == '18'
+      - ansible_distribution_version == '18.04'
+    quiet: yes
+
+- name: Check if node is installed
+  command: /bin/bash -c "source /home/{{ ansible_user }}/.nvm/nvm.sh; node -v | cut -c 2-"
+  register: node_check
+  failed_when: "'command not found' in node_check.stdout"
+
+- debug:
+    msg: "[WARNING] Current node version is lower than recommended version {{ node_version }}"
+  when: node_check.stdout != '' and node_check.stdout is version_compare(node_version, '<')
+
+- name: Check if npm is installed
+  command: /bin/bash -c "source /home/{{ ansible_user }}/.nvm/nvm.sh; npm -v"
+  register: npm_check
+  failed_when: "'command not found' in npm_check.stdout"
+
+- debug:
+    msg: "[WARNING] Current npm version is lower than recommended version {{ npm_version }}"
+  when: npm_check.stdout != '' and npm_check.stdout is version_compare(npm_version, '<')
+
+- name: Install Node and NPM if node or npm command not found
+  include_tasks:
+    file: install.yml
+  when: node_check.stdout == '' or npm_check.stdout == ''
```

Comment:
If an older version of node is installed but no npm is installed, this will invoke the install script, correct? We should only install if neither is already installed.

Reply:
Changed to and condition